app/controllers/v3/service_instances_controller.rb

```diff
@@ -131,6 +132,7 @@ def share_service_instance
     service_instance = ServiceInstance.first(guid: hashed_params[:guid])
     resource_not_found!(:service_instance) unless service_instance && can_read_service_instance?(service_instance)
     unauthorized! unless can_write_space?(service_instance.space)
+    suspended! unless permission_queryer.is_space_org_active?(service_instance.space.guid)
```

Comment:
Add tests to spec/request/service_instances_spec.rb:
- post '/service_instances'
- post '/service_instances/:guid/relationships/shared_spaces'
- patch '/service_instances/:guid'
- delete '/service_instances/:guid'
- delete '/service_instances/:guid/relationships/shared_spaces/:space_guid'